php/php.editor/src/org/netbeans/modules/php/editor/completion/PHPCodeCompletion.java

```diff
@@ -1424,6 +1434,158 @@ public boolean isAccepted(PhpElement element) {
         }
     }
 
+    private void autoCompleteClassMethodParameterName(
```

Comment:
Based on this method : https://github.com/apache/netbeans/blob/4e3035d755449849d352439fedc5baf9cfd40ca3/php/php.editor/src/org/netbeans/modules/php/editor/completion/PHPCodeCompletion.java#L1289